src/char/int_guild.cpp

```diff
@@ -859,14 +880,14 @@ int guild_calcinfo(struct guild *g)
 
 	if(g->guild_lv<=0)
 		g->guild_lv = 1;
-	nextexp = guild_nextexp(g->guild_lv);
+	nextexp = guild_exp_db.get_nextexp(g->guild_lv);
 
 	// Consume guild exp and increase guild level
 	while(g->exp >= nextexp && nextexp > 0){	//fixed guild exp overflow [Kevin]
 		g->exp-=nextexp;
 		g->guild_lv++;
 		g->skill_point++;
-		nextexp = guild_nextexp(g->guild_lv);
+		nextexp = guild_exp_db.get_nextexp(g->guild_lv);
 	}
 
 	// Save next exp step
```

Comment:
Cant make a suggestion here, but these two lines could be removed with the changes above.